Engines/Wine/Settings/Font smoothing/script.js

```diff
@@ -0,0 +1,82 @@
+include("engines.wine.engine.object");
+include("engines.wine.plugins.regedit");
+include("engines.wine.plugins.font_smoothing");
+
+/**
+ * setting to set the Fonts Smoothing
```

Comment:
Multiline comments like class and method comments should start with an Uppercase letter.

Reply:
Do we really do it like that in other places?

Reply:
In Java I am, in JavaScript I don't think so yet. The reason for starting with an uppercase letter is because I believe this is normal for normal sentences, right? Class and method descriptions are normally converted to a HTML API documentation document, which consists of ... normal sentences.

Reply:
I don't know. Is there an official guideline regarding this?

Reply:
I don't know of a guideline, but I can give you some examples:
- JavaScript (typescript): https://github.com/angular/angular/blob/978ffa9d32017e72eba400d01707c8fbc68aa25e/packages/core/src/interface/lifecycle_hooks.ts
- Java: https://github.com/AdoptOpenJDK/openjdk-jdk11/blob/master/src/java.base/share/classes/java/lang/Thread.java

Reply:
We should document this somewhere.

Reply:
See #1023